src/dataviews/index.ts

```diff
@@ -0,0 +1,6 @@
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews/wp';
+
+window.LockedPrivateDataViews = {
+	filterSortAndPaginate,
+	DataViews,
+};
```

Comment:
It would probably be good to put a comment in this file about why it's needed.

Reply:
Done in https://github.com/Automattic/remote-data-blocks/pull/416/commits/e7564f6b91bd80156be2dac3be42982505695098